content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -42,6 +46,8 @@ cd:
   enabled: true
 ----
 
+In your PR towards the RPU, include a link to the PR in your plugin, which contains all the necessary changes, like described above.
```

Comment:
I would not shorten this as people will be unlikely to be familiar with this acronym, (I realise it is declared above but I think this is clearer)
```suggestion
In your PR towards the repository permission updater, include a link to the PR in your plugin, which contains all the necessary changes, like described above.
```